tests/Iterator/CsvJsonFileIteratorTest.php

```
@@ -13,6 +13,30 @@
  */
 class CsvJsonFileIteratorTest extends TestCase
 {
+    /**
+     * @expectedException        \InvalidArgumentException
+     * @expectedExceptionMessage Cannot open file for reading: this_file_is_not_existed
+     */
+    public function testConstructorWithFileIsNotExisted()
+    {
+        $iterator = new CsvJsonFileIterator('this_file_is_not_existed');
+    }
+
+    /**
+     * @expectedException        \RuntimeException
+     * @expectedExceptionMessage Syntax error
+     */
+    public function testReadRowShouldReturnRuntimeException()
```

Please use Throw instead of Return